monocle/db/queries.py

```diff
@@ -759,3 +764,44 @@ def new_contributors(es, index, repository_fullname, params):
     )
     diff = new.difference(old)
     return {'items': [n for n in new_authors if n['key'] in diff]}
+
+
+def changes_by_file_map(es, index, repository_fullname, params):
+    params = deepcopy(params)
+    params['etype'] = ("Change",)
+    body = {
+        "size": 1000,
+        "query": generate_filter(repository_fullname, params),
+    }
+    data = run_query(es, index, body)
+    changes = [r['_source'] for r in data['hits']['hits']]
+    files = {}
+    for change in changes:
+        for f in change['changed_files']:
+            key = '{}:{}'.format(change['repository_fullname'], f['path'])
+            try:
+                files[key] += 1
+            except KeyError:
+                files[key] = 1
+    return {'changes': files}
+
+
+def authors_by_file_map(es, index, repository_fullname, params):
```

Comment:
Do you think we could get the change rate by file for each authors in the returned values ?

Reply:
we'll have to define what it really means but I'd prefer to that in another PR.